### PR TITLE
update index.html

### PR DESCRIPTION
**Pull request by zeesh97**
is par mene title change kia hai.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
     <link rel="stylesheet" type="text/css" href="style.css">
 
-    <title>Saylani Welfare Trust!</title>
+    <title>Saylani Welfare Trust! zeeshan updated</title>
```
Comment (zaidbinahmed) [inline index.html:19]: ye line me ghalat change kiya he

**Review by zaidahmed332675 — APPROVED** [review]
approved

File: index.html (hunk 1 above)
Comment (zaidbinahmed) [reply index.html:19]: han dikha

Comment (zeesh97) [reply index.html:19]: han dekh chuka

**Pull request merged**